test/integration/last_test.rb

```diff
@@ -7,8 +7,10 @@
     end
   end
 
+  # The DistrictAdmin resource does not support the last method.
   resources = [Clever::District, Clever::School, Clever::Student,
-               Clever::Event, Clever::Section, Clever::Teacher]
+               Clever::Event, Clever::Section, Clever::Teacher,
+               Clever::SchoolAdmin]
```

Comment:
Yes, because the DistrictAdmin tests would fail since your district admin API does not support the limit parameter.